### PR TITLE
Adds AppInsightsTelemetryConsumer, new app-insights export subpath and README updates.

### PR DESCRIPTION
**Pull request by seanimam**
## Description
- Adds the concrete implementation of ITelemetryConsumer, `AppInsightsTelemetryConsumer` to the package and exports it under a new subpath `/app-insights`
-  Flips README xample 1 and 2 around so the Azure App insights example is first
- README.md updated with usage of new AppInsightsTelemetryConsumer

### CODE REVIEW

File: packages/framework/client-logger/fluid-telemetry/README.md [hunk 1]
```diff
@@ -45,59 +35,59 @@ if (containerExists) {
...
+// Done! Your container telemetry is now being created and sent to your Telemetry Consumer which will forward it to Azure App Insights.
 ```
 
-## Example 2: Logging container telemetry to Azure App Insights
+Congrats, that's it for now! If you've decided to use Azure App Insights, we have designed useful prebuilt queries for you that utilize the generated telemetry
 
-Before you can get telemetry sent to Azure App Insights, you'll need to create an Instance of App Insights on Azure. Then you'll be able to create an Azure App Insights client that you can easily turn into a ITelemetryConsumer and finally hook it up to container telemetry. [Learn more about Azure App Insights](https://learn.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview)
+**It's important to note that while you can create your own custom implementation of `ITelemetryConsumer` for sending telemetry to App Insights, if you change the shape of the telemetry emitted from our implementation, `AppInsightsTelemetryConsumer`, we make no guarentess of our provided App Insights queries or dashboards working as expected.**
```
Comment (alexvy86) [inline packages/framework/client-logger/fluid-telemetry/README.md:66]: Let's remove this bit and add it when the queries are here?

File: packages/framework/client-logger/fluid-telemetry/README.md [hunk 2]
```diff
@@ -45,59 +35,59 @@ if (containerExists) {
...
 
-### Step 1: Install the @fluidframework/fluid-telemetry package and Azure App Insights package dependencies
+## Example 2: Extending ITelemtryConsumer to send Fluid telemetry to a custom destination
```
Comment (alexvy86) [inline packages/framework/client-logger/fluid-telemetry/README.md:68]: Typo
```suggestion
## Example 2: Extending ITelemetryConsumer to send Fluid telemetry to a custom destination
```

File: packages/framework/client-logger/fluid-telemetry/README.md [hunk 3]
```diff
@@ -112,42 +102,23 @@ if (containerExists) {
     myAppContainerId = await myAppContainer.attach();
 }
 
-// 2: This is our implementation of ITelemetryConsumer that will send telemetry to Azure App Insights
-class AppInsightsTelemetryConsumer implements ITelemetryConsumer {
-    constructor(private readonly appInsightsClient: ApplicationInsights) {}
-
+// 2: This is our implementation of ITelemetryConsumer
```
Comment (alexvy86) [inline packages/framework/client-logger/fluid-telemetry/README.md:105]: Probably remove section 2 here, since Step 2 above has this same class definition?

**Review by alexvy86 — APPROVED** [review]
Left a few comments below but overall LGTM

File: packages/framework/client-logger/fluid-telemetry/README.md (hunk 3 above)
Comment (seanimam) [reply packages/framework/client-logger/fluid-telemetry/README.md]: I see how that might be confusing. I've updated the example to be as if you're imported the implementation from another file.

**Pull request merged**